src/components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import {metrics} from '../helpers/metrics'
+import type {ITheme} from '../theme'
+import type {ViewProps} from 'react-native'
+
+export type CardProps = {
+  onPress?: () => void
+  children?: React.ReactNode
+  bgColor?: string
+  activeOpacity?: number
+  style?: ViewProps
+}
+
+const Card: React.FC<CardProps> = ({onPress, children, bgColor, style}) => (
+  <View onPress={onPress} activeOpacity={onPress ? 0.8 : 1} bgColor={bgColor} style={style}>
```

Comment:
define constant for number

Reply:
yes, updated. Please help me check again